editoast/src/client/mod.rs

```diff
@@ -60,12 +60,12 @@ pub struct Client {
     pub command: Commands,
 }
 
-#[derive(ValueEnum, Debug, Derivative, Clone)]
-#[derivative(Default)]
+#[derive(ValueEnum, Debug, Educe, Clone)]
+#[educe(Default)]
```

Comment:
It's kind of a new feature in rust, but now when you want to define a **simple unit variant** as the default value of an enum. You can use the standard macro `Default`. See doc [here](https://doc.rust-lang.org/std/default/trait.Default.html#derivable).

```suggestion
#[derive(ValueEnum, Debug, Default, Clone)]
```